sbncode/CAFMaker/FillReco.cxx

```diff
@@ -844,6 +869,21 @@ namespace caf
             p.mult = h->Multiplicity();
             p.start = h->StartTick();
             p.end = h->EndTick();
+
+
+            // Get the trajectory point index from this hit. Again -- this is too hard. 
+            //
+            // Use this to get the (SCE corrected) efield and the angle to the drift direction
+            unsigned traj_point_index = thms.at(i_hit)->Index();
+            unsigned int int_max_as_unsigned_int{std::numeric_limits<int>::max()};
+            if (traj_point_index != int_max_as_unsigned_int && // invalid
+                track.HasValidPoint(traj_point_index)) {
```

Comment:
According to [`HasValidPoint()`](https://code-doc.larsoft.org/docs/latest/html/classrecob_1_1TrackTrajectory.html#a2ef083f5e80a3c9b6a4db02c36b0ba68), if the index is not valid, `false` is returned.
Since we can be reasonably sure that the value checked here (`std::numeric_limits<int>::max()`... where have you found it?) is never a valid point (its track would take at least 64 GB of memory), you can collapse this check to:
```suggestion
            if (track.HasValidPoint(traj_point_index)) {
```

Reply:
Done!